docs/kyma/04-02-local-installation.md

```diff
@@ -7,9 +7,10 @@ This Installation guide shows you how to quickly deploy Kyma locally on the MacO
 
 ## Prerequisites
 
+- [Kyma CLI](https://github.com/kyma-project/cli) 
 - [Docker](https://www.docker.com/get-started)
-- [Minikube](https://github.com/kubernetes/minikube) 0.33.0
-- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.12.0
+- [Minikube](https://github.com/kubernetes/minikube) 1.0.1
```

Comment:
Do we really need 1.0.1 specifically, or are we fine with 1.0 in general? Maybe we don't need to indicate the minor version here?